packages/asc-ui/src/components/Modal/Modal.tsx

```diff
@@ -81,7 +81,11 @@ function Modal({
           }
         : {})}
     >
-      <BackDrop backdropOpacity={backdropOpacity} onClick={handleClose} />
+      <BackDrop
+        backdropOpacity={backdropOpacity}
+        onClick={handleClose}
+        {...otherProps}
```

Comment:
Passing all other props to `BackDrop` might cause issues. If you overwrite the style of the `Modal` component for example, this style overwrite would affect both `ModalStyle` and `BackDrop`. I'd pass `zIndexOffset` explicitly to both `BackDrop` and `ModalStyle`

Reply:
Thanks for the feedback. I will update the PR accordingly.